### PR TITLE
Corrected link to Packaged Apps Documentation

### PR DESCRIPTION
**Pull request by sakshamsaxena**
Fixed Issue #1361 
Peace

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ This will output a package and output to ```package/archives/```. You can use
 WebIDE to install this package in the device or simulator.
 
 Further details can be found in the
-[Marketplace documentation](http://marketplace.readthedocs.org/latest/topics/package.html).
+[Marketplace documentation](http://marketplace.readthedocs.org/en/latest/topics/packaged-apps.html).
```
Comment (atiqueahmedziad) [inline README.md]: you should not write the `en`

Comment (sakshamsaxena) [inline README.md]: I understand. I too thought about this, but I believe this is the page. Don't see other language versions of it. I'm about to start reading the docs. Sorta new to moz

Comment (atiqueahmedziad) [inline README.md]: Oh ! sorry , its alright :)

**Comment by mstriemer** [comment]
Not sure why the `en` portion of the URL is required but it sure seems to be. Thanks for the fix!

**Pull request merged**